### DISCUSSION TITLE
Azure & Visual Studio publishing

### DISCUSSION THREAD

**Discussion by donros** [chkr1011/mqttHub#8] [Q&A]
Hi, what is the best way to publish this MQTT broker to MS Azure using Visual Studio?

**Comment by chkr1011** [comment]
There is no way since this only a library and no stand alone implementation with an ASP host etc. Please check out mqttHub. It is a standalone MQTT broker based on this library. You can publish it as an App Service to Azure.

**Reply by donros** [reply]
Thank you!

**Comment by oubingbing** [comment]
你好，邮件已收到，谢谢。
